impl/createApi.go

```diff
@@ -228,15 +244,13 @@ func createAndDeployOpenAPI(openAPISpec openapi3.T, filePath, namespace, service
 	var rule utils.Rule
 	var backendRef utils.BackendRef
 
-	var serviceUrlArr []string
-
-	if serviceUrl != "" {
-		serviceUrlArr = strings.Split(serviceUrl, ".")
-	} else {
+	if serviceUrl == "" {
+		var serviceUrls []string
 		for _, serverEntry := range openAPISpec.Servers {
-			serviceUrlArr = append(serviceUrlArr, serverEntry.URL)
+			serviceUrls = append(serviceUrls, serverEntry.URL)
```

Comment:
If we are going to select the first one, why do we need to maintain an array of URLs?

Reply:
As a temp approach we have used the first url, but in future we may have to follow a better approach to pick the more valid one depending on a given usecase. For now let's store it? WDYT?